lighthouse-core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -61,4 +61,21 @@ describe('Resources are fetched over http/2', () => {
       assert.ok(auditResult.displayValue === '');
     });
   });
+
+  it('passes when all resources were requested via service worker', () => {
```

Comment:
Should we also get a test for failing with mixed records. i.e. only convert 1 or 2 1.x requests to serviceWorkers so that it isn't all serviceWorker requests (so it fails), so that it's more realistic.

Reply:
sure I just looked at the test of h2 and copied it, I can scope to 1 or 2 :)

Reply:
to @exterkamp's point, the tests are currently passing yet our test case is mutating `fromServiceWorker` and our code is checking `fetchedViaServiceWorker`, so something isn't quite right :)

Reply:
`fromServiceWorker` is what devtools has in it's network trace but we mutate it to fetchedViaServiceWorker. 

see https://github.com/GoogleChrome/lighthouse/blob/1098f952d5c75b22668aa572532415f81d2817af/lighthouse-core/lib/network-request.js#L260

it's also from devtools:
https://github.com/ChromeDevTools/devtools-frontend/blob/ecf32a28e6156b8a2c9dc025faa9315ec8652e6a/front_end/sdk/NetworkManager.js#L337-L338

Reply:
ahhhhhhh that makes more sense it's doing the devtoolslog roundtrip!

@wardpeet think you could fix the bug over in network-records-to-devtools-log? it's reading the wrong property name when doing the `networkRecordsToDevtoolsLog` round-trip :)